<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mount GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/mount-gui.png</normaloff>:/icons/mount-gui.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QStatusBar {
  border-color: rgb(159, 157, 154);
  border-style: inset;
  border-width: 1px;
  margin-left: 1px;
};</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="BlkListWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="autoExpandDelay">
       <number>-1</number>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="itemsExpandable">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>100</number>
      </attribute>
      <column>
       <property name="text">
        <string>Device</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>FS type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Label</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mountpoint</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>UUID</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin" stdset="0">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Device:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="MntDeviceEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>128</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Monospace</family>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Filesystem Type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="FsTypeEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>128</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Monospace</family>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Mountpoint:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MountpointEdit">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Target path for mount.&lt;br/&gt;No character escaping required.&lt;/p&gt;&lt;pre style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'monospace'; font-weight:700;&quot;&gt;Following placeholders are recognized:&lt;br/&gt;%d - short device name, e.g. sda1 (sanitized)&lt;br/&gt;%D - sanitized device path, e.g. dev_sda1&lt;br/&gt;%u - current user name.&lt;br/&gt;%g - current user group.&lt;br/&gt;%t - filesystem type (lowercase).&lt;br/&gt;%l - sanitized LABEL.&lt;br/&gt;%L - LABEL as-is.&lt;br/&gt;%U - UUID as-is.&lt;br/&gt;&lt;br/&gt;Empty LABEL or UUID will be substituted by %d.&lt;/pre&gt;&lt;p&gt;Sanitizing functions turns any non-alphanumeric unicode characters other than '-', '_' and '.' to '_', and removes any repeating punctuation and punctuation at the start and the end of the string.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelMountOptions">
           <property name="text">
            <string>Mount options: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelHelpLink">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Open integrated manual for the filesystem</string>
           </property>
           <property name="text">
            <string>[&lt;a href=&quot;qrc:/help/mount.html&quot;&gt;available options&lt;/a&gt;]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="MntOptionsEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Mount -o options. &lt;/p&gt;
&lt;p&gt;Same placeholders are available as for mountpoint.&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="UnmountButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string>Unmount</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="MountButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string>Mount</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuUseSystemd">
     <property name="title">
      <string>Use Systemd</string>
     </property>
     <addaction name="actionUse_systemctl"/>
     <addaction name="actionUse_systemd_umount"/>
    </widget>
    <addaction name="actionSave_by_FS"/>
    <addaction name="actionSave_by_Label"/>
    <addaction name="actionSave_by_UUID"/>
    <addaction name="separator"/>
    <addaction name="actionMakeFstabEntry"/>
    <addaction name="actionMakeSystemdUnit"/>
    <addaction name="separator"/>
    <addaction name="menuUseSystemd"/>
    <addaction name="separator"/>
    <addaction name="actionEditPreferences"/>
   </widget>
   <addaction name="menuPreferences"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionNetworkScan"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QLabel" name="StatusBarLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;big&gt;[&lt;a href=&quot;#&quot;&gt;Log&lt;/a&gt;]&lt;/big&gt; </string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <property name="indent">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
   </property>
   <property name="statusTip">
    <string>Update list of block devices (Ctrl+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionMount">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Mount</string>
   </property>
  </action>
  <action name="actionUnmount">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Unmount</string>
   </property>
   <property name="toolTip">
    <string>Unmount given mountpoint</string>
   </property>
  </action>
  <action name="actionSave_by_FS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-save-as.png</normaloff>:/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save by FS type</string>
   </property>
   <property name="toolTip">
    <string>Save by FS type</string>
   </property>
   <property name="statusTip">
    <string>Save preferences for the given filesystem type.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_by_Label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-save-as.png</normaloff>:/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save by Label</string>
   </property>
   <property name="toolTip">
    <string>Save by Label</string>
   </property>
   <property name="statusTip">
    <string>Save preferences for the given partition label.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_by_UUID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-save-as.png</normaloff>:/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save by UUID</string>
   </property>
   <property name="toolTip">
    <string>Save by UUID</string>
   </property>
   <property name="statusTip">
    <string>Save preferences for the given UUID.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEditPreferences">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-properties.png</normaloff>:/icons/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Edit Preferences</string>
   </property>
   <property name="statusTip">
    <string>Open configuration file in the text editor.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionMakeSystemdUnit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/emblem-system.png</normaloff>:/icons/emblem-system.png</iconset>
   </property>
   <property name="text">
    <string>Make Systemd Unit</string>
   </property>
   <property name="statusTip">
    <string>Create systemd mount unit file.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionUse_systemd_umount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use systemd-umount</string>
   </property>
   <property name="statusTip">
    <string>Use systemd-umount when possible.</string>
   </property>
  </action>
  <action name="actionUse_systemctl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use systemctl</string>
   </property>
   <property name="statusTip">
    <string>Use systemctl when a mount unit exists for the target.</string>
   </property>
  </action>
  <action name="actionMakeFstabEntry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/emblem-system.png</normaloff>:/icons/emblem-system.png</iconset>
   </property>
   <property name="text">
    <string>Make FSTAB Entry</string>
   </property>
   <property name="statusTip">
    <string>Add an entry to the fstab file.</string>
   </property>
  </action>
  <action name="actionNetworkScan">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/nmap.png</normaloff>:/icons/nmap.png</iconset>
   </property>
   <property name="statusTip">
    <string>Scan local network for shared disks (Ctrl-N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>labelHelpLink</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>ShowHelp(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMount</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionMount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUnmount</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionUnmount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_by_FS</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionSave_by_FS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_by_Label</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionSave_by_Label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_by_UUID</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionSave_by_UUID()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditPreferences</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionEditPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BlkListWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>OnBlkListSelection(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StatusBarLabel</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>ShowLog(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMakeSystemdUnit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionMakeSystemdUnit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MntDeviceEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>OnMntDeviceChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUse_systemctl</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>OnUseSystemdChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUse_systemd_umount</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>OnUseSystemdChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMakeFstabEntry</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionMakeSystemdUnit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>onShow()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNetworkScan</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnActionNetworkScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>onShow()</signal>
  <slot>ShowHelp(QString)</slot>
  <slot>ShowLog(QString)</slot>
  <slot>OnActionRefresh()</slot>
  <slot>OnActionMount()</slot>
  <slot>OnActionUnmount()</slot>
  <slot>OnActionSave_by_FS()</slot>
  <slot>OnActionSave_by_UUID()</slot>
  <slot>OnActionSave_by_Label()</slot>
  <slot>OnActionEditPreferences()</slot>
  <slot>OnActionMakeSystemdUnit()</slot>
  <slot>OnBlkListSelection(QTreeWidgetItem*)</slot>
  <slot>OnMntDeviceChanged(QString)</slot>
  <slot>OnUseSystemdChanged(bool)</slot>
  <slot>OnSettingsChanged()</slot>
  <slot>OnActionNetworkScan()</slot>
 </slots>
</ui>
